File: src/app/shared/models/user.model.ts
export class UserModel {
    DisplayName?: string;
    Firstame?: string;
    LastName?: string;
    UserId: string;
    Email?: string;
    JobTitle?: string;

    constructor(userId?: string, first?: string, last?: string, display?: string
        , email?: string, job?: string) {
        this.UserId = userId;
        this.DisplayName = display;
        this.Firstame = first;
        this.LastName = last;
        this.Email = email;
        this.JobTitle = job;
    }
}

File: WebAPI/AquaChatAPI.Domain/Models/UserModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AquaChatAPI.Domain.Models
{
    public class UserModel
    {
        public string UserId { get; set; }
        public string[] Email { get; set; }
        public string JobTitle { get; set; }
        public string DisplayName { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Language { get; set; }
    }
}
